### ISSUE TITLE
Standardize and synchronize possible columns in DataFrame

### ISSUE THREAD

**Issue by EwoutH** [EwoutH/shipping-data#29]
TODO

**Comment by EwoutH** [comment]
@thomassandbergen and @imvs95, maybe we could tackle this together on our meeting next Monday. The earlier we do this, the less current data, webscrapers, scripts and documentation need to be modified.

**Comment by imvs95** [comment]
Good idea, let's!

**Comment by thomassandbergen** [comment]
Good idea, hopefully the Maersk scraper will  also have data in Pandas then
